### DISCUSSION TITLE
Do notes titles matter a lot ?

### DISCUSSION THREAD

**Discussion by avichou** [logancyang/obsidian-copilot#434] [Q&A]
long story short im trying to decide whether to setup automatic backup from upnote , or to use the export function

the main difference between the two lies in the filenames: auto backup is UUID , and export (manual export) has the proper filenames and titles 

here's an example of UUID titles ; 0a2ced31-1230-4b50-882e-9bebe04c73c5 

my question is, is the title important when it comes to searching for answers and asking questions to your vault ? or is it not rly that bad so long as the actual note and all the content is properly rendered. basically only the title looks bad 

<img width="787" alt="image" src="https://github.com/brianpetro/obsidian-smart-connections/assets/86660671/711d3f29-7c37-4140-8361-4f5490ae4a9b">

**Comment by logancyang** [comment]
Short answer: the title matters. 

Long answer: copilot chunks the notes and each chunk has the original note's title, the vector search is on title + chunk. After retrieved, LLM sees the context with the title. So the title matters to some extent.
